tutorials/docusaurus.config.js

```diff
@@ -199,6 +199,23 @@ module.exports = {
         ],
       },
     ],
+    [
+      '@iota-wiki/plugin-tutorial',
+      {
+        title: 'Connect to Shimmer EVM Testnet and deploy a Solidity Smart Contract',
+        description:
+          'In this tutorial, we will connect to Shimmer EVM Testnet, connect Metamask to it, get funded and deploy a smart contract in Solidity.',
+        preview: 'connect-shimmerevm-testnet-and-deploy.png',
+        route: 'tutorials/shimmerevm-setup',
+        tags: [
+          'text',
+          'shimmer',
+          'solidity',
+          'getting-started',
+          'wasp'
```

Comment:
```suggestion
        tags: [
          'text',
          'shimmer',
          'solidity',
          'getting-started',
          'wasp',
          'iscp'
```

Nearly forgot that one 😅